procedures/unit-testing-utils.ipf

```diff
@@ -78,4 +78,70 @@ static Function/S TextWaveToList(txtWave, sep)
 	return list
 End
 
+#if IgorVersion() > 7.0
```

Comment:
After our insight to int64/uint64 comparison logic in Igor, I think these functions are now obsolete?
If so please remove them.